packages/lib-user/dev/components/App/App.js

```diff
@@ -147,40 +154,42 @@ function App({
       theme={zooTheme}
       themeMode={dark ? 'dark' : 'light'}
     >
-      <main>
-        <header>
+      <main className={adminMode ? 'admin-mode' : ''}>
+        <header aria-label='Zooniverse site header'>
           <AuthModal
             activeIndex={activeIndex}
             closeModal={closeAuthModal}
             onActive={setActiveIndex}
           />
-          <h1><a href="./">lib-user</a></h1>
-          {user?.login ? (
-            <button onClick={logout}>
-              Logout
-            </button>
-          ) : (
-            <button onClick={login} disabled={loading}>
-              Login
-            </button>
-          )}
-          <label>
-            Dark Theme
-            <input
-              name='theme-toggle'
-              onChange={() => setDarkTheme(!dark)}
-              type="checkbox"
-              value={!dark}
-            >
-            </input>
-          </label>
+          <ZooHeader
+            adminMode={adminMode}
+            onThemeChange={() => setDarkTheme(!dark)}
+            register={openRegisterModal}
+            showThemeToggle
+            signIn={openSignInModal}
+            signOut={onSignOut}
+            themeMode={dark ? 'dark' : 'light'}
+            user={user || {}}
+          />
         </header>
         {loading ? 
           <p>Loading...</p> : (
           <div>
             {content}
           </div>
         )}
+        <footer>
+          <p>⚠️ WARNING ⚠️ : this is lib-user, links in the components above are intended to be used in app-root and may not work as expected in lib-user</p>
```

Comment:
While the ZooHeader and ZooFooter do display while running lib-user locally, what's the reasoning behind adding them to the dev app?

Reply:
Mostly to look better/similar to what the pages will actually look like, the convenience of the footer admin toggle, but the ZooHeader and ZooFooter are not necessary.

Reply:
Gotcha ~ overall I think you should include components that make development of lib-user an easy experience, but I also think it's worth considering that lib-user as a concept (a library of React components used in /stats and /groups pages) does not include direct interaction with ZooHeader or ZooFooter. If you do decided to keep ZooHeader and ZooFooter in the dev app, I recommend placing the 'warning' text directly below the `<header>` before the main content, and specifying "this is a dev app - links in the header and footer will not work as expected".